dashboard_examples/plotly-dash/README.md

```diff
@@ -0,0 +1,36 @@
+The code for example apps is found here: https://github.com/plotly/dash-sample-apps
```

Comment:
The first thing we should say here is that this example can be deployed via cds dashboards on nebari using the default `dashboard` environment that ships with nebari. 

Alternately, you can run it individually using the environment.yml file in this folder.